src/olympia/files/tests/test_utils_.py

```diff
@@ -244,22 +244,37 @@ def test_summary(self):
         assert (
             self.parse({'description': 'An addon.'})['summary'] == 'An addon.')
 
+    def test_invalid_app_versions_are_ignored(self):
+        """Invalid versions are ignored."""
+        data = {
+            'applications': {
+                'gecko': {
+                    # Not created, so are seen as invalid.
+                    'strict_min_version': '>=30.0',
+                    'strict_max_version': '=30.*',
+                    'id': '@random'
+                }
+            }
+        }
+        assert not self.parse(data)['apps']
+
     def test_apps_use_provided_versions(self):
         """Use the min and max versions if provided."""
-        firefox_min_version = self.create_appversion('firefox', '30.0')
-        firefox_max_version = self.create_appversion('firefox', '30.*')
+        firefox_min_version = self.create_appversion('firefox', '47.0')
+        firefox_max_version = self.create_appversion('firefox', '47.*')
+
         self.create_webext_default_versions()
         data = {
             'applications': {
                 'gecko': {
-                    'strict_min_version': '>=30.0',
-                    'strict_max_version': '=30.*',
```

Comment:
I re-added those, I just adapted the version numbers to represent something more reasonable since we only support WebExtensions in more recent firefox versions and this tests only tests that the version numbers are accepted. We now fail if the `strict_min_version` is below the min version for webextensions.